ooniprobe/Base.lproj/Main.storyboard

```diff
@@ -383,8 +383,19 @@
                                                 <color key="textColor" red="0.46666666670000001" green="0.46666666670000001" blue="0.46666666670000001" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
                                                 <nil key="highlightedColor"/>
                                             </label>
+                                            <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" textAlignment="center" lineBreakMode="tailTruncation" numberOfLines="0" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="qUW-JG-UiH">
+                                                <rect key="frame" x="16" y="217" width="288" height="50"/>
+                                                <constraints>
+                                                    <constraint firstAttribute="height" constant="50" id="iaj-Yv-BPx"/>
+                                                </constraints>
+                                                <string key="text">ooniprobe: 1.0
```

Comment:
OK. To avoid confusion you should probably rename it to something that clearly indicates it's a placeholder and if for some reason it's not replaced we can more easily spot it. For example changing it to X.Y.Z.